### PR TITLE
fix: List with actions style should be the right

### LINKED ISSUE #16232: The actions are not at the far right of List.Item
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/embed/n7k2k6kj)

### Steps to reproduce
You can compare it with the [3.14.1 version](https://codesandbox.io/s/1q0vkxwqy3)

### What is expected?
The actions are at the far right of List.Item

### What is actually happening?
The actions are at the bottom of List.Item

| Environment | Info |
|---|---|
| antd | 3.16.4 |
| React | 16.4.0 |
| System | mac os 10.14.1 |
| Browser | chrome 73.0.3683.103 |

---
It is ok below 3.14.1 version 
https://codesandbox.io/s/1q0vkxwqy3

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

fix #16232 

### 💡 Solution

Add `float: right` to the actions.

### 📝 Changelog

Fix List with actions style not be the right

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
Not sure if its the correct answer. Please help to check @afc163

**Comment by afc163** [comment]
可以用这个排查一下 https://github.com/ant-design/ant-design/issues/15329

File: components/list/style/index.less [hunk 1]
```diff
@@ -206,6 +202,15 @@
...
   }
+
+  // ============================ without flex ============================
+  &-item-no-flex {
+    display: block;
+
+    .@{list-prefix-cls}-item-action {
+      float: right;
+    }
+  }
```
Comment (afc163) [inline components/list/style/index.less]: 好像没必要从 &-item 里移出来？

Comment (zombieJ) [reply components/list/style/index.less]: 按功能划分为了以后找和处理起来方便

**Pull request merged**